pkg/codegen/main.go

```diff
@@ -147,6 +155,19 @@ func main() {
 	if err := replaceCoreControllerImports(); err != nil {
 		panic(err)
 	}
+	// Comment out this function to avoid adding the test session import in the management controllers
```

Comment:
Definitely a nit question; maybe I don't get this bit, do we need to comment out the lines? Files are going to be deleted and this will need to happen on the newly generated files right?

Reply:
The other team started this comment pattern when they introduced the replaceImport function. It can be taken out. It doesn't matter much to me.